src/main/java/com/imgix/URLBuilder.java

```diff
@@ -31,8 +31,15 @@ public URLBuilder(String domain, boolean useHttps, String signKey, boolean inclu
 		this.includeLibraryParam = includeLibraryParam;
 	}
 
+	/**
+	 * This single-parameter constructor, accepts a domain string and
+	 * returns a `URLBuilder` with it's `useHttps` member variable set
+	 * to `true` by default.
+	 *
+	 * @param domain - a valid domain string, e.g. `example.imgix.net`
+	 */
```

Comment:
Also, if we're going to add doc comments for one of the constructors, it only feels right to add it for all other public methods as well.

Reply:
This is true, and desired. I will document everything else in a separate PR. I wanted to keep this change scoped appropriately.

Reply:
Great, in that case let's remove it here for cleanliness and include it in your separate PR.